### PR TITLE
Add doc check in CI

### PR DESCRIPTION
**Pull request by polarker**

### CODE REVIEW

**Comment by polarker** [comment]
@simerplaha Could you help to fix the format issue of doc? Thanks

**Review by tdroxler — APPROVED** [review]
LGTM, let's merge after the  one from @simerplaha that fix that

File: .github/workflows/scala.yml
Comment (tdroxler) [inline .github/workflows/scala.yml:30]: actually this will be `unidoc` after the Simer's merge

**Pull request merged**
